diff --git a/node_modules/web-push/src/vapid-helper.js b/node_modules/web-push/src/vapid-helper.js
index c0305bd..ffdab16 100644
--- a/node_modules/web-push/src/vapid-helper.js
+++ b/node_modules/web-push/src/vapid-helper.js
@@ -77,7 +77,7 @@ function validateSubject(subject) {
 
   let subjectParseResult = null;
   try {
-    subjectParseResult = new URL(subject);
+    subjectParseResult = new window.URL(subject);
   } catch (err) {
     throw new Error('Vapid subject is not a valid URL. ' + subject);
   }
@@ -195,7 +195,7 @@ function getVapidHeaders(audience, subject, publicKey, privateKey, contentEncodi
   }
 
   try {
-    new URL(audience); // eslint-disable-line no-new
+    new window.URL(audience); // eslint-disable-line no-new
   } catch (err) {
     throw new Error('VAPID audience is not a url. ' + audience);
   }
